### PR TITLE
DX-1669: Add ide:info command

### PR DESCRIPTION
**Pull request by danepowell**
@anavarre before I add tests, how do you feel about this?

### CODE REVIEW

File: src/Command/Ide/IdeInfoCommand.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    $this->io->definitionList(
+      ['IDE property' => 'IDE value'],
+      new TableSeparator(),
+      ['UUID' => $response->uuid],
+      ['Label' => $response->label],
+      ['Owner name' => $response->owner->first_name . ' ' . $response->owner->last_name],
+      ['Owner username' => $response->owner->username],
+      ['Owner email' => $response->owner->mail],
+      ['Cloud application' => $response->links->application->href],
+      ['Editor URL' => $response->links->ide->href],
```
Comment (grasmash) [inline src/Command/Ide/IdeInfoCommand.php:49]: Let’s remain consistent and call this the IDE url

**Comment by danepowell** [comment]
The test failure is unrelated, I'll research and fix it separately.

**Comment by danepowell** [comment]
Tests fixed, it seems Chocolatey is having reliability issues: https://github.com/chocolatey/choco/issues/1629

**Comment by anavarre** [comment]
Tried to checkout the PR to test the new command but I'm not seeing it yet. Let me know when it's okay to test.

**Comment by danepowell** [comment]
@anavarre commands are cached, did you clear the kernel cache? `acli ckc`

**Comment by anavarre** [comment]
Didn't know about `acli ckc`. That helped, thanks.

Manual testing:

```
$ ./bin/acli ide:info
Please select a Cloud Platform application:
  [0] mytestapp
 > 0
Please select an IDE to get more information:
  [0] mytestide
 > 0
 ------------------- -------------------------------------------------------------------------------- 
  IDE property        IDE value                                                                       
 ------------------- -------------------------------------------------------------------------------- 
  UUID                <REDACTED>                                            
  Label               test                                                                            
  Owner name          Aurelien Navarre                                                                
  Owner username      anavarre                                                                        
  Owner email         <REDACTED>                                                     
  Cloud application   https://cloud.acquia.com/api/applications/<REDACTED>  
  IDE URL             https://<REDACTED>.ide.ahdev.cloud                    
  Web URL             https://<REDACTED>.web.ahdev.cloud                    
 ------------------- --------------------------------------------------------------------------------
```

**Review by anavarre — APPROVED** [review]

**Review by grasmash — APPROVED** [review]

**Pull request merged**